src/main/java/com/autotune/database/dao/ExperimentDAO.java

```diff
@@ -94,4 +94,6 @@ public interface ExperimentDAO {
     // Load metadata
     List<KruizeDSMetadataEntry> loadMetadata() throws Exception;
 
+    // Delete metadata
+    public ValidationOutputData deleteKruizeDSMetadataEntryByName(String dataSourceName);
```

Comment:
Perhaps you could name it something like "removeClusterMetadataEntryByName" or

Reply:
Have followed the similar naming convention as `deleteKruizeExperimentEntryByName`, please confirm if this still requires renaming.